docs/static/playground/playground.js

```diff
@@ -1,404 +1,266 @@
-                '};',
-            ].join('\n'), 'require.d.ts');
-
-            var loading = document.getElementById('loading');
-            loading.parentNode.removeChild(loading);
-            load();
-
-        });
-    };
-
-    var editor = null;
-    var data = {
-        js: {
-            model: null,
-            state: null
-        },
-        css: {
-            model: null,
-            state: null
-        },
-        html: {
-            model: null,
-            state: null
-        }
-    };
-
-    function load() {
-
-        function layout() {
-            var GLOBAL_PADDING = 20;
-
-            var WIDTH = window.innerWidth - 2 * GLOBAL_PADDING;
-            var HEIGHT = window.innerHeight;
-
-            var TITLE_HEIGHT = 110;
-            var FOOTER_HEIGHT = 80;
-            var TABS_HEIGHT = 20;
-            var INNER_PADDING = 20;
-            var SWITCHER_HEIGHT = 30;
-
-            var HALF_WIDTH = Math.floor((WIDTH - INNER_PADDING) / 2);
-            var REMAINING_HEIGHT = HEIGHT - TITLE_HEIGHT - FOOTER_HEIGHT - SWITCHER_HEIGHT;
-
-            playgroundContainer.style.width = WIDTH + 'px';
-            playgroundContainer.style.height = (HEIGHT - FOOTER_HEIGHT) + 'px';
-
-            sampleSwitcher.style.position = 'absolute';
-            sampleSwitcher.style.top = TITLE_HEIGHT + 'px';
-            sampleSwitcher.style.left = GLOBAL_PADDING + 'px';
-
-            typingContainer.style.position = 'absolute';
-            typingContainer.style.top = GLOBAL_PADDING + TITLE_HEIGHT + SWITCHER_HEIGHT + 'px';
-            typingContainer.style.left = GLOBAL_PADDING + 'px';
-            typingContainer.style.width = HALF_WIDTH + 'px';
-            typingContainer.style.height = REMAINING_HEIGHT + 'px';
-
-            tabArea.style.position = 'absolute';
-            tabArea.style.boxSizing = 'border-box';
-            tabArea.style.top = 0;
-            tabArea.style.left = 0;
-            tabArea.style.width = HALF_WIDTH + 'px';
-            tabArea.style.height = TABS_HEIGHT + 'px';
-
-            editorContainer.style.position = 'absolute';
-            editorContainer.style.boxSizing = 'border-box';
-            editorContainer.style.top = TABS_HEIGHT + 'px';
-            editorContainer.style.left = 0;
-            editorContainer.style.width = HALF_WIDTH + 'px';
-            editorContainer.style.height = (REMAINING_HEIGHT - TABS_HEIGHT) + 'px';
-
-            if (editor) {
-                editor.layout({
-                    width: HALF_WIDTH - 2,
-                    height: (REMAINING_HEIGHT - TABS_HEIGHT) - 1
-                });
+var _a, _b;
+// @ts-ignore
+var STORAGE_KEY = "SAVED_SAMPLE";
+var ENDPOINT_KEY = "SAVED_ENDPOINT";
+var existing = localStorage.getItem(STORAGE_KEY);
+// @ts-ignore
+var editor = monaco.editor.create(document.getElementById("container"), {
+    value: existing || "",
+    language: "typescript",
+    minimap: { enabled: false }
+});
+var targets = [
```

Comment:
I've also refactored our list of editors in the 'editors.json" file, which you can fetch. WOuld be good to eventually unify all these tools to use the same list of editors.